packages/esm-patient-registration-app/src/patient-registration/input/input.scss

```diff
@@ -116,3 +118,7 @@
   margin: 0;
   padding-left: 1rem;
 }
+
+.Input {
+  max-width: 300px;
```

Comment:
giving all the inputs a large size

Reply:
I remember we already have the max-width placed somewhere, can you please find that first?
Thanks!

Reply:
I see we were setting the max-width for the whole form [here](https://github.com/openmrs/openmrs-esm-patient-management/blob/main/packages/esm-patient-registration-app/src/patient-registration/patient-registration.scss#L32) in teh `infoGrid` class. Here am trying to set the max-width for the  inputs